docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -0,0 +1,40 @@
+= Legacy Promise API removed from Keycloak JS
+
+With this release, we have removed the legacy Promise API methods from Keycloak JS. This means that calling `.success()` and `.error()` on promises returned from the adapter is no longer possible. Instead standardized Promise methods such as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then[`.then()`] and https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch[`.catch()`] should be used.
+
+*Before:*
```

Comment:
Sorry, I should have told you before. Release notes should contain only a brief description of what was changed. Details on what migration steps are needed should be mentioned in [upgrading docs](https://github.com/keycloak/keycloak/blob/main/docs/documentation/upgrading/topics/keycloak/changes-22_0_0.adoc).

See, for example, this PR to the old docs repository: https://github.com/keycloak/keycloak-documentation/pull/1760/files